parser/parser.y

```diff
@@ -2448,6 +2451,7 @@ FunctionNameConflict:
 |	"UTC_DATE"
 |	"CURRENT_DATE"
 |	"VERSION"
+|	"INTERVAL" %prec lowerThanIntervalKeyword
```

Comment:
`INTERVAL` is a [reserved keyword](https://dev.mysql.com/doc/refman/5.7/en/keywords.html) in MySQL, for reserved keyword, you need add it [here](https://github.com/pingcap/tidb/blob/master/parser/parser.y#L54) , [here](https://github.com/pingcap/tidb/blob/master/parser/parser.y#L2091) and test it [here](https://github.com/pingcap/tidb/blob/master/parser/parser_test.go#L60)

This is because reserved keyword can't be used as table name, database name and so on, not the same as normal identifier.